templates/manifest.json

```diff
@@ -200,6 +200,22 @@
                 "memoryMbytes": 2048,
                 "timeoutSecs": 3600
             }
+        },
+        {
+            "id": "js-cypress",
+            "name": "project_cypress",
+            "label": "Cypress",
+            "category": "javascript",
+            "technologies": [
+                "nodejs"
+            ],
+            "description": "Empty starter actor. A blank slate with minimal starting code for developing actors from scratch using JavaScript.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/feat/add-cypress-actor-template/dist/templates/js-cypress.zip?raw=true",
```

Comment:
This needs to be switched back to master branch, using other branch name is just for local testing.